tests/flows/basic_erc20_send_receive.go

```diff
@@ -88,19 +93,15 @@ func BasicERC20SendReceive(network interfaces.Network) {
 		fundedKey,
 	)
 
-	// Relay the message to subnet B and check for message delivery
+	// Relay the message to Subnet A and check for message delivery
 	receipt = network.RelayMessage(
 		ctx,
 		receipt,
+		cChainInfo,
 		subnetAInfo,
-		subnetBInfo,
 		true,
 	)
 
-	messageExecutedEvent, err := teleporterUtils.GetEventFromLogs(receipt.Logs, subnetBInfo.TeleporterMessenger.ParseMessageExecuted)
```

Comment:
Was there a specific reason to remove these "message executed" checks?

Reply:
I don tthink message executed checks should be required in all Teleporter app e2e test checks. If we wanted to check message executed, I'd opt to add it in Teleporter's `RelayMessage`. Also, if it doesn't execute the message properly, the rest of the test would fail.